receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -785,3 +793,30 @@ metrics:
       value_type: int
     attributes: [index_aggregation_type]
     enabled: false
+  elasticsearch.index.segments.count:
+    description: Number of segments of an index.
+    unit: "{segments}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [index_aggregation_type]
+    enabled: true
```

Comment:
We need to have these disabled by default going forward, otherwise it is a breaking change and requires use of a feature gate.